src/turtle/env/Dmq.d

```diff
@@ -321,10 +331,13 @@ public class Dmq : Node!(DmqNode, "dmq")
 
     ***************************************************************************/
 
-    override public NodeItem node_item ( )
+    override public AddrPort node_addrport ( )
     {
-        assert(this.node);
-        return this.node.node_item;
+        AddrPort addrport;
+        addrport.setAddress(this.node.node_item.Address);
+        addrport.port = cast(ushort)this.node.node_item.Port;
```

Comment:
Why cast? https://github.com/sociomantic-tsunami/swarm/blob/v4.x.x/src/swarm/Const.d#L89

Reply:
Oh. Well, I copied it from here: https://github.com/sociomantic-tsunami/dlsproto/blob/v14.x.x/src/turtle/env/Dls.d#L235

Reply:
Ha.